mewarpx/mewarpx/diags_store/particle_diagnostic.py

```diff
@@ -0,0 +1,151 @@
+"""Diagnostic code that wraps the picmi.ParticleDiagnostics class"""
+
```

Comment:
In the future when renaming files, I think we should all use `git mv old_name new_name`. That way, git can continue to keep track of the change history in the file. If you rename the file outside of git mv, git thinks you deleted the old file and created a new one, which basically throws out all of the old tracked changes.